src/pages/cart-page/CartPage.css

```diff
@@ -0,0 +1,160 @@
+.ha-cart-content > * {
+  padding: 0.625rem;
+  text-align: center;
+}
+.ha-cart-content {
+  position: relative;
+  margin: 0 auto;
+  display: grid;
+  grid-gap: 0.625rem;
+  grid-template-columns: 1fr;
+  grid-template-rows: min-content auto min-content;
+}
+.ha-cart-gtr {
+  grid-column: 1/2;
+  grid-row: 2/3;
+  background-color: var(--ha-light-background-color);
+}
+.cart-heading {
+  padding: 1rem;
+}
+.h-cart-container {
+  justify-content: center;
+
+  display: flex;
+  align-items: flex-start;
+  gap: 0.625rem;
+}
+.h-cart-container {
+  justify-content: center;
+  display: flex;
+}
+.card-wrap {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+.h-imagecart-wrapper {
+  /* margin: 0 2rem; */
+  /* width: 80%; */
+  display: flex;
+  background-color: var(--ha-white-color);
+}
+
+.h-cart-quantity {
+  display: flex;
+}
+
+.image-cart img {
+  width: 100%;
+}
+.image-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  margin: auto;
+  /* padding: 1rem 2rem 0 2rem; */
+}
+.image-text h3,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-bottom: 0.3rem;
+}
+.image-cart-btn {
+  display: flex;
+  flex-direction: column;
+  padding-top: 0.5rem;
+}
+.image-cart-btn button {
+  padding: 0.5rem;
+  margin-top: 0.5rem;
+}
+.remove-btn {
+  cursor: pointer;
+  background-color: var(--ha-primary-color);
+  color: var(--ha-white-color);
+  border: none;
+}
+.move-btn {
+  cursor: pointer;
+  background-color: var(--ha-white-color);
+  color: var(--ha-primary-color);
+  border: 0.063rem solid var(--ha-primary-color);
+}
+.move-btn:hover {
+  color: var(--ha-white-color);
+  background-color: var(--ha-primary-color);
+}
+.h-price-box {
+  width: 30%;
+  background-color: var(--ha-white-color);
+  padding: 1.5rem;
+  line-height: 3rem;
+}
+.item-pricee {
+  display: flex;
+  flex-direction: row;
+}
+.total-price {
+  font-weight: bold;
+}
+.item-pricee p:last-child {
+  margin-left: auto;
+}
+.cart-price {
+  font-weight: bold;
+}
+.blank-div {
+  border: 1px solid var(--ha-gray-color);
+}
+.price-btn {
+  width: 100%;
+  padding: 0.5rem;
+  background-color: var(--ha-primary-color);
+  color: var(--ha-white-color);
+}
+
+@media only screen and (max-width: 800px) {
```

Comment:
put media queries at end as good practice

Reply:
it is in end only.